Makefile

```diff
@@ -84,8 +85,14 @@ else
 TESTCMD = -run $(RUNTEST)
 endif
 
+test_skip_kube: SKIP_KUBE=1
+test_skip_kube: test
+
 test:
-	@go test ./... -p 1 -parallel=16 $(TESTCMD) -count=1
+	@export PATH=$(PATH):$(PWD); \
```

Comment:
Wouldn't we want to invert the precedence here to always prefer the build artifact?